docs/source/user-guide/concepts/conda-performance.rst

```diff
@@ -0,0 +1,106 @@
+
+Therefore, if you're experiencing a slowdown, evaluate the following questions
+to identify potential causes:
+
+* Are you creating a new environment or installing into an existing one?
+* Does your environment have pip-installed dependencies in it?
+* What channels are you using?
+* What packages are you installing?
+* Is the channel metadata sane?
+* Are channels interacting in bad ways?
+
+
+
+Improving conda performance
+===========================
+
+To address these challenges, you can move packages to archive
+channels and follow the methods below to present a reduced view of
+all available packages.
+
+To speed up conda, we offer the following recommendations.
+
+Are you:
+    * Using conda-forge?
+        * Use conda-metachannel to reduce conda’s problem size.
+    * Using bioconda?
+        * Use conda-metachannel to reduce conda’s problem size.
+        * `Read more about docker images <https://github.com/bioconda/bioconda-recipes/issues/13774>`_.
+    * Specifying very broad package specs?
+        * Be more specific. Letting conda filter more candidates makes it faster.
+          For example, instead of ``numpy``, we recommend ``numpy=1.15`` or, even better, ``numpy=1.15.4``.
+        * If you are using R, instead of specifying only ``r-essentials``, specify ``r-base=3.5 r-essentials``.
+    * Feeling frustrated with “verifying transaction” and also feeling lucky?
+        * Run ``conda config –set safety_checks disabled``.
+    * Getting strange mixtures of defaults and conda-forge?
+        * Run ``conda config –set channel_priority strict``.
+        * This also makes things go faster by eliminating possible mixed solutions.
+    * Observing that an Anaconda or Miniconda installation is getting slower over time?
+        * Create a fresh environment. As environments grow, they become harder
+          and harder to solve. Working with small, dedicated environments can
+          be much faster.
+
+Set strict channel priority
+---------------------------
+
+Setting strict channel priority makes it so that if a package exists on
+a channel, conda will ignore all packages with the same name on lower
+priority channels.
+
+.. figure:: ../../img/strict-disabled.png
+    :width: 50%
+.. figure:: ../../img/strict-enabled.png
+    :width: 50%
+
+This can dramatically reduce package search space and reduces the use of
+improperly constrained packages.
+
+One thing to consider is that setting strict channel priority may make
+environments unsatisfiable. Learn more about :ref:`strict`.
+
+
+Reduce the index
+----------------
+One option for speeding up conda is to reduce the index. It's likely that
+your repodata contains package data that is not used in the solving stage.
+Filtering out unnecessary packages before solving can save time.
+Reducing the index takes specs and reduces the problem size so that the
+SAT doesn’t have as much work to do.
+
+Additionally, it is beneficial to be as specific as possible in your package
```

Comment:
Reducing the index is something that conda does based on the user's input specs.  Making your inputs more specific makes this index reduction more effective, and thus makes the process faster.  Users alone do not reduce the index, but they should understand why it's important to think about specs.